packages/plugins/sentry/src/index.ts

```diff
@@ -164,7 +169,7 @@ export const useSentry = (options: SentryPluginOptions = {}): Plugin => {
                   childSpan.setData('result', result);
                 }
 
-                if (result instanceof Error && !(result instanceof EnvelopError)) {
+                if (result instanceof Error && !(result instanceof EnvelopError) && !skipError(result)) {
```

Comment:
shouldn't the default value for `skipError` be `() => result instanceof EnvelopError`?

Reply:
That is a good idea.

Reply:
Actually, I don't think that will work as there is internal use of `EnvelopError` which should be ignored regardless of what users have defined with `skipError`.

Reply:
If people wanna customize the behavior and want to report certain `EnvelopError` instances they need to change the behavior. I think it is totally fine that people need to add this check manually in case they want to customize stuff. We can even export a `defaultSkipError` function that is `(result) => result instanceof EnvelopError`, so people can re-use it.

Reply:
I don't really understand what `EnvelopError` instances users would care to report I assume there is a good reason to skip sending them which is why it's in place. Do you have an example scenario in mind? it's not totally clear to me.

For my purposes I wouldn't need to change that behaviour. I just want to skip errors that I've defined.

Maybe if you can elaborate on the idea though that might help me understand what you're getting at a bit better.

Reply:
`EnvelopError` is used by the `useMaskedErrors` plugin (https://github.com/dotansimha/envelop/blob/83c27fbceab0dd83377cc5cf344e26292facd656/packages/core/src/plugins/use-masked-errors.ts#L15-L23).  Any error that does not extend `EnvelopError` will be replaced with a GraphQLError with the message `Unexpected error.`.

There might be cases, where people want to report `EnvelopError`s, thus we should keep the flexibility for users to opt into fully controlling what they want to report.